app/javascript/react/src/containers/Home.js

```diff
@@ -9,24 +9,54 @@ class Home extends Component {
   constructor(props) {
     super(props)
     this.state = {beers:[
-      {beerName: "Bud Light",
+      {name: "Bud Light",
       brewery: "Busch",
-      rating: 2, id: 1}, {beerName: "Bud",
+      rating: 2, id: 1}, {name: "Bud",
       brewery: "Busch",
-      rating: 2, id: 2}]
+      rating: 2, id: 2}],
+      search_value: ''
     }
```

Comment:
Are you at the point where you can avoid hard coding this? 
If you wish for a default, simply move this data to a constant and import it in as props to this component.